cleancode/src/main/java/com/epam/cleancode/comments/InvalidInputException.java

```diff
@@ -1,16 +1,16 @@
 package com.epam.cleancode.comments;
 
 public class InvalidInputException extends RuntimeException{
-    String message;
-    public InvalidInputException(String s) {
-        super();
-        this.message = s;
 
+    private final String message;
```

Comment:
please go to Throwable and reuse field from this class and pass exceptionMessage to super. This field should be removed